Sigma_specification.md

```diff
@@ -355,7 +363,11 @@ Creator of the rule. (can be a name, nickname, twitter handle...etc)
 References to the source that the rule was derived from. These could be blog articles, technical papers, presentations or even tweets.
 
 ## Date (optional)
-
+/// I think of something like
+date:
+    creation: YYYY/MM/DD
+    modified: YYYY/MM/DD
+///
```

Comment:
My idea was more to gather temporal information in a single `date` field.
```yaml
date:
    creation: YYYY-MM-DD
    modified: YYYY-MM-DD
```
But I think there's been a discussion since then and it hasn't been a consensus.

Reply:
I dislike the structure for two reasons:
- you have an additional line --> worse oversight of the whole file
- if you don't have a modified date it looks like:
```
date:
    creation: 2023-06-16
```